mantra/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mantrachain",
```

Comment:
since this is a testnet, please move this entire directory into the /testnets/ directory, and then rename this directory and the chain_name to either "mantratestnet" or "mantrachaintestnet" (must be consistent)

Reply:
Done.